src/main/java/com/sap/sgs/phosphor/fosstars/data/github/GitHubDataFetcher.java

```diff
@@ -39,6 +39,7 @@
 /**
  * Helper class for GitHub data providers which pulls the data from a GitHub repository. Also, the
  * class caches the fetched data for a certain amount of time.
+ * <p>The class is thread-safe.</p>
```

Comment:
Let's please use HTML markup.

```suggestion
 * <p>Helper class for GitHub data providers which pulls the data from a GitHub repository. Also, the
 * class caches the fetched data for a certain amount of time.</p>
 * <p>The class is thread-safe.</p>
```